Generate Compass dependencies artifact (#8690)
kyma-project/kyma: installation/resources/README.md

| `installer-cr-cluster.yaml.tpl` | Provides complete cluster Kyma installation. | ✅ |
| `installer-cr-cluster-with-compass.yaml.tpl` | Provides complete cluster Kyma installation with the `compass` and `compass-runtime-agent` components. These components will eventually become part of complete Kyma installation and this configuration will be deleted.  | ⛔️ |
| `installer-cr-cluster-compass.yaml.tpl` | Provides cluster Compass installation only with the Kyma components that Compass uses. | ⛔️ |

> | `installer-cr-cluster-compass-dependencies.yaml.tpl` | Provides cluster Kyma installation only with the Kyma components that Compass uses (without Compass component). | ⛔️ |

Reviewer: ```suggestion
| `installer-cr-cluster-compass-dependencies.yaml.tpl` | Provides cluster Kyma installation only with the Kyma components that are essential for Compass integration. | ⛔️ |
```
